templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -11,9 +11,10 @@ contents:
       dst_path="/etc/NetworkManager/system-connections"
       if [ -d $src_path ]; then
         echo "$src_path exists"
-        fileList=$(echo {br-ex,ovs-if-br-ex,ovs-port-br-ex,ovs-if-phys0,ovs-port-phys0}.nmconnection)
+        # NetworkManager 1.15.2 or later now requires that files in /{etc,run}/NetworkManager/system-connections end in the suffix '.nmconnection'
+        fileList=$(echo {br-ex,ovs-if-br-ex,ovs-port-br-ex,ovs-if-phys0,ovs-port-phys0} {br-ex,ovs-if-br-ex,ovs-port-br-ex,ovs-if-phys0,ovs-port-phys0}.nmconnection)
         for file in ${fileList[*]}; do
-          if [ ! -f $dst_path/$file ]; then
+          if [ ! -f $dst_path/$file ] && [ -f $src_path/$file ]; then
```

Comment:
@pliurh is this really necessary? if the file we wrote is not there, then this should be a failure

Reply:
It's because `fileList` contains keyfiles with and without the extension '.nmconnection', e.g. 'br-ex' and 'br-ex.nmconnection'. So to make it work for both RHEL and RHCOS, I add the source file check here.

Reply:
I understood the file extension differences between the two env but I am not clear on why we need src file check still 
not sure in when we don't have the src file ?

Reply:
In RHEL we don't have src files with the extension, in RHCOS we don't the src files without the extension.